_strategies/prototyping.md

```diff
@@ -24,37 +24,35 @@ The following principles are meant to guide product teams in the use of technica
 
-For example, consider using a boilerplate app (or simple demo app) to support development of a Git merging / branching strategy for a project, or to build out a [DevOps pipeline](/devops), particularly for clients that do not have a well-developed DevOps culture.
+For example, consider using a boilerplate app (or simple demo app) to support development or to build out a minimal [DevOps pipeline](/devops), can help you see where the bottlenecks to agile development will likely occur before a vendor is brought on.
 
 ### Help vendors, don’t handcuff them
 
-An acquisition prototype can serve as a reference implementation for a vendor, by demonstrating one approach to technical issues like data access or authentication.
+A prototype can serve as a reference implementation for a vendor, by demonstrating one approach to technical issues like data access or authentication.
 
-An acquisition prototype should not dictate a choice of specific technology or approach (e.g., one language over another), unless there are compelling reasons to do so. But a prototype should adhere to the technology requirements of the client (e.g., if a client’s IT standards dictate Microsoft .NET platform as the preferred technology stack, the prototype should be built on this stack).
+A prototype should not dictate a choice of specific technology or approach (e.g., one language over another), unless there are compelling reasons to do so. But a prototype should adhere to the technology requirements of the client (e.g., if a client’s IT standards dictate Microsoft .NET platform as the preferred technology stack, the prototype should be built on this stack).
 
 ### Use all the tools in risk mitigation toolbox
 
-A prototype should minimize risk prior to procurement, but probably won’t eliminate it. There may be questions unanswered or risks present when issuing an RFP — that’s normal. RFPs themselves should have some provisions for managing risk (particularly if they are smaller in scope), and using an agile methodology can provide opportunities to change course as issues arise after contract award.
+A prototype should reduce risk prior to procurement, but probably won’t eliminate it. There may be questions unanswered or risks present when issuing an RFP — that’s normal. RFPs themselves should have some provisions for managing risk (particularly if they are smaller in scope), and using an agile methodology can provide opportunities to change course as issues arise after contract award.
 
 ### Help the buyer develop prototyping capacity
 
-Be sure to assess the capability of the client to continue prototype work after the engagement is over. If you’re not helping the buyer develop their own capacity for using prototypes, you’re probably doing it wrong.
```

Comment:
This was one of our findings from the AK Medicaid project. Just curious why you eliminated it.

Reply:
"You're probably doing it wrong" is not a particularly constructive way to frame this finding. I think we could keep it in if we discuss how that can introduce more risk into the project going forward.

Reply:
I massaged the language a bit and added to the preceding paragraph.

Reply:
The only thing is this is the only page that kind of talks about the "client" as another entity, rather than a lot of the other pages that speak to "you" or at least the state/government- wonder if we want to reframe in what you added?

Reply:
Yeah, good point. This was adapted from another source and it still shows that framing.